brainscore_core/supported_data_standards/brainio/upload_validator.py

```diff
@@ -101,7 +102,7 @@ def validate_assembly(assembly, identifier="assembly") -> bool:
         raise ValidationError(f"{identifier}: Missing 'coords' attribute")
     
     # Check for stimulus_id coordinate
-    if 'stimulus_id' not in assembly.coords:
+    if hasattr(assembly, 'stimulus_id') :
```

Comment:
```suggestion
    if not hasattr(assembly, 'stimulus_id') :
```

This should have a `not` because we want `stimulus_id` should be part of the coordinate.